Documentation/Maintenance/Release.md

```diff
@@ -799,54 +783,15 @@ Release Notes Posts
    git shortlog --topo-order --no-merges v$old_version..v$new_version
 ```
 
-TODO: update for ITK
-
-In `Utilities/Maintenance/release`, there are the following scripts:
-
-  * `make-changelog.sh`
-  * `prep-emails.py`
-  * `send-emails.sh`
-
-First, create the `changes.txt` file by giving a commit range to
-`make-changelog.sh`:
-
-```sh
-   make-changelog.sh $previous_release $release
```

Comment:
This part sounds useful.

Reply:
We already have a method to generate the changelog (above). These are left over from the VTK migration.